File: JAVA-CarRacingProject/ProiectCurse/src/main/java/domain/Participant.java
package domain;

public class Participant extends Entity<Integer> {
    private String nume;
    private String echipa;

    public Participant(String nume, String echipa){
        this.nume=nume;
        this.echipa=echipa;

    }

    public String getNume() {
        return nume;
    }

    public String getEchipa() {
        return echipa;
    }


    public void setNume(String nume) {
        this.nume = nume;
    }

    public void setEchipa(String echipa) {
        this.echipa = echipa;
    }

    @Override
    public String toString() {
        return "Participant{"+
                "nume="+this.nume+
                " echipa="+this.echipa+
                "}";
    }


}

File: C#-CarRacingProject/Domain/Participant.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ProiectCSharpVersion.Domain
{
    class Participant:Entity<int>
    {
        public string Nume { get; set; }
        public string Echipa { get; set; }
        public Participant(string nume, string echipa)
        {
            this.Nume = nume;
            this.Echipa = echipa;
        }

        public override string ToString()
        {
            return "Participat: nume=" + Nume + " echipa=" + Echipa;
        }
    }
}
